src/main/java/core/basesyntax/dao/StorageDao.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Fruit;
+import java.util.Map;
+
+public interface StorageDao {

```

Comment:
let's fix naming, it's should be FruitDao since you're working with Fruit model here